en/protocol/camera.md

```diff
@@ -2,47 +2,201 @@
 
 The camera protocol allows to configure camera payloads and request their status. It supports photo and video cameras and includes messages to query and configure the onboard camera storage.
```

Comment:
"allows to" is not correct - should be something like "The camera protocol is used to configure ..."